lib/nimble_template/addons/test_interactive.ex

```diff
@@ -0,0 +1,28 @@
+defmodule NimbleTemplate.Addons.TestInteractive do
+  @moduledoc false
+
+  use NimbleTemplate.Addons.Addon
+
+  @impl true
+  def do_apply(%Project{} = project, _opts) do
+    Generator.inject_mix_dependency({:mix_test_interactive, "~> 1.2", only: :dev, runtime: false})
+
+    add_dev_config(project)
+  end
+
+  defp add_dev_config(project) do
+    Generator.inject_content(
```

Comment:
Can we try with `Generator.append_content` 🤔

Reply:
Updated here: https://github.com/nimblehq/elixir-templates/pull/247/commits/911faa558c4937349e2fbcb4982c09e3d5fb04c6